src/lib/taro/types.ts

```diff
@@ -2,8 +2,11 @@
  * Shared types to normalize interfaces between the different
  * Taro implementations
  */
-
-export interface TaroAsset {
+export interface TaroAssetCommon {
+  balance?: string;
+  amount?: string;
+}
+export interface TaroAsset extends TaroAssetCommon {
```

Comment:
This makes the type a bit more flexible for components that handle TaroAssets and Tarobalances.